src/components/UserCard.js

```diff
@@ -9,11 +9,26 @@ export default class UserCard extends Component {
         this.state={
             userData:{},
             userName:"ravi771999",
+            profileExists:true,
         }
     }
 
     async componentDidMount(){
-        let response=await axios.get(`https://api.github.com/users/${this.state.userName}`);
+        let response=null;
+
+        try{
+            response = await axios.get(`https://api.github.com/users/${this.state.userName}`);
+        }catch (err) {
+            
+        }
+
+        if(response == null){
+            this.setState({
+                profileExists:false,
+            })
+            return;
+        }
```

Comment:
create the util for api call. See what does the axios returns and also read about promised. Try and Catch block should not be used.